pkg/cmd/agent-task/create/create.go

```diff
@@ -52,11 +71,20 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 			}
 			return createRun(opts)
 		},
+		Example: heredoc.Doc(`
+			# Create a task from an inline description
+			$ gh agent-task create "build me a new app"
+
+			# Create a task from a file
+			$ gh agent-task create -F task-desc.md
+		`),
 	}
 	if f != nil {
 		cmdutil.EnableRepoOverride(cmd, f)
 	}
 
+	cmd.Flags().StringVarP(&fromFileName, "from-file", "F", "", "Read task description from file")
```

Comment:
As I'm taking `pr create` as a reference command that supports reading from a file:
- We should mention the use of `-` to read from stdin
- We should enclose "file" in the usage string in backticks to make it render as the arg name in the `--help` output. I mean like this in `pr create` docs (note the "file" next to "--body-file"):
  ```
  -F, --body-file file       Read body text from file (use "-" to read from standard input)
  ```

If you're up for the changes, here's the change:

```suggestion
	cmd.Flags().StringVarP(&fromFileName, "from-file", "F", "", "Read task description from `file` (use \"-\" to read from standard input)"
```